e2e/test/exit-prematurely-dry-run-fails/verify/verify.ts

```diff
@@ -1,6 +1,5 @@
-import { promises as fs } from 'fs';
+import { promises as fsPromises } from 'fs';
 import chai, { expect } from 'chai';
-import { it } from 'mocha';
```

Comment:
I think thats an issue

Reply:
Importing `it` from mocha is optional. Mocha will make sure `it` is globally available. I've removed it since we're never importing `it`.